.ci/cico_build.sh

```diff
@@ -30,9 +30,13 @@ if [[ "$DeveloperBuild" != "true" && "$PR_CHECK_BUILD" != "true" && "$USE_CHE_LA
 
   set -x
   yum -y update
-  yum -y install centos-release-scl java-1.8.0-openjdk-devel git patch bzip2 golang docker
+  yum -y install centos-release-scl java-1.8.0-openjdk-devel git patch bzip2 golang
```

Comment:
This command can be replaced by `yum -y install java-1.8.0-openjdk-devel patch bzip2 golang`. The `patch` and `golang` is used in cico_utils.sh too, but it does not have its own method. Maybe it would be worth creating a method for that in cico_utils.sh and reusing here?

and using those methods from cico_utils.sh:
```
installScl
installGit
```

Reply:
yeah, I believe it is a good idea for further refactoring ;)
I just scratched the surface in this PR

Reply:
Ok :)